components/multiselect/multiselect.jsx

```diff
@@ -39,6 +39,10 @@ export default class MultiSelect extends React.Component {
         loading: PropTypes.bool,
     }
 
+    static defaultProps = {
+        valueKey: 'id',
```

Comment:
It wasn't immediately obvious to me why this fixed the issue until I looked at the component more closely. I wonder if it would be beneficial to note here in a comment why we default to 'id'?